pallets/committee/src/lib.rs

```diff
@@ -380,6 +389,42 @@ pub mod pallet {
 
             Ok(().into())
         }
+
+        #[pallet::weight(10_000)] // TODO: Set weights
+        /// Set new constituent members
+        ///
+        /// This call can only be triggered by `fn propose_constituents` momentary
+        /// after approving of `propose_constituents`
+        pub fn set_members(origin: OriginFor<T>) -> DispatchResultWithPostInfo {
+            ensure_root(origin)?;
+            Ok(().into())
+        }
+
+        #[pallet::weight(10_000)] // TODO: Set weights
+        /// Propose new constituents to the constituent committee
+        ///
+        /// * This call only can be called by `MemberType::Council`
+        /// * This call will propose the call `set_members`
+        pub fn propose_constituents(
+            origin: OriginFor<T>,
+            _constituents: Vec<AccountIdFor<T>>,
+        ) -> DispatchResultWithPostInfo {
+            let _selector = Self::ensure_member(origin);
+            Ok(().into())
```

Comment:
I think we can leverage the existing `propose` extrinsic and simply call this here with `::add_constituent` extrinsic as the action.

Reply:
The problem is it can't be wrapped into an `Action` directly, see https://github.com/ChainSafe/PINT/pull/63/commits/f3fde30c5cb27fb44f2a8745eb30f822d4a5ae92